qa-tests-backend/src/test/groovy/ReconciliationTest.groovy

```diff
@@ -183,14 +183,14 @@ class ReconciliationTest extends BaseSpecification {
 
         // Recreate sensor
         try {
-            orchestrator.createOrchestratorDeployment(sensorOrchestratorDeployment)
+            orchestrator.scaleDeployment("stackrox", "sensor", 1)
         } catch (Exception e) {
             log.error("Error re-creating the sensor: ", e)
             throw e
```

Comment:
since we rehtrow the error maybe we can remove `try catch` as it only adds logging that will be logged anyway when we throw

Reply:
Done